ckanext/switzerland/templates/admin/manage_showcase_admins.html

```diff
@@ -0,0 +1,74 @@
+{% extends "admin/base.html" %}
+
+{% import 'macros/form.html' as form %}
+
+{% set user = c.user_dict %}
+
+{% block primary_content_inner %}
+  <h1 class="page-heading">
+    {% block page_heading %}{{  _('Manage Application Admins') }}{% endblock %}
+  </h1>
+  {% block form %}
+    <form class="dataset-form form-horizontal add-member-form" method='post'>
+      <div class="row-fluid">
+        <div class="control-group control-medium">
+          <label class="control-label" for="username">
+            {{ _('Add an Existing User') }}
+          </label>
+          <span>
+            {{ _('To make an existing user a Application Admin, search for their username below.') }}
+          </span>
+
+          <div class="controls">
+            <input id="username" type="text" name="username" placeholder="{{_('Username')}}"
+            value="" class="control-medium" data-module="autocomplete"
+            data-module-source="/api/2/util/user/autocomplete?q=?">
+          </div>
+        </div>
+      </div>
+
+      <div class="form-actions">
+        <button class="btn btn-primary" type="submit" name="submit" >
+          {{ _('Add User') }}
+        </button>
+      </div>
+    </form>
+  {% endblock %}
+
+  <h3 class="page-heading">{{ _('Application Admins') }}</h3>
+  {% if c.showcase_admins %}
+  <table class="table table-header table-hover table-bordered">
+    <thead>
+      <tr>
+        <th scope="col">{{ _('User') }}</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for user_dict in c.showcase_admins %}
+      <tr>
+        <td class="media">
+          {{ h.linked_user(user_dict['id'], maxlength=20) }}
+          {% set locale = h.dump_json({'content': _('Are you sure you want to remove this user from the Application Admin list?')}) %}
+          <div class="btn-group pull-right">
+            <a class="btn btn-danger btn-small" href="{% url_for controller='ckanext.showcase.controller:ShowcaseController', action='remove_showcase_admin', user=user_dict['id'] %}" data-module="confirm-action" data-module-i18n="{{ locale }}" title="{{ _('Remove') }}">{% block delete_button_text %}<i class="icon-remove"></i> {{ _('Remove') }}{% endblock %}</a>
+          </div>
+        </td>
+      </tr>
+      {% endfor %}
+    </tbody>
+  </table>
+  {% else %}
+    <p>{{ _('There are currently no Application Admins.') }}</p>
+  {% endif %}
+{% endblock %}
+
+{% block secondary_content %}
+  {{ super() }}
+  <div class="module module-narrow module-shallow">
+    <div class="module-content">
+      {% trans %}
+         <p><strong>Application Admin:</strong> Can create and remove applications. Can add and remove datasets from applications.</p>
```

Comment:
I just noticed that this is not translateable. Is that on purpose?

Reply:
The `{% trans %}` tag makes multi-line strings translateable. :)